app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class Api::V1::ProjectsController < ApplicationController
+  before_action :authorize_request
+  before_action :project_find, only: %i[show update destroy]
+
+  def index
+    @projects = @current_user.projects.order(created_at: :desc)
+
+    render json: ProjectSerializer.new(@projects).serialized_json, status: :ok
+  end
+
+  def show
+    render json: ProjectSerializer.new(@project).serialized_json, status: :ok
+  end
+
+  def create
+    @project = @current_user.projects.new(project_params)
```

Comment:
Do we need `@` for `@projects` here? I think it could be common variable without `@`